_posts/blog/2020-01-23-2020-01-23-lab-meeting.md

```diff
@@ -0,0 +1,173 @@
+  These projects don't track success if trainees leave academia 😭
+* Professional services can support you.....but they're often not promoted or enabled.
+* Don't look for a specific mentor, build a mentor from LOTS of different people and their strengths.
+* If the goals of supervisor and student are not aligned then they could try to design outputs that are going to benefit both.
+  For example if the student wants to go into industry, they could focus on building infrastructure and pipelines in the lab rather than writing papers.
+  (Although this can backfire when it comes to the degree requirements - at least at the moment!)
+
+**Sarah** asked how to encourage people to use a service?
+Hub23 (the Turing's private BinderHub) doesn't currently have any users which is a shame because she's put a lot of work into making it really easy for me to maintain (e.g. [writing bots](https://github.com/HelmUpgradeBot) to keep the helm chart up-to-date and the container registry in a manageable state).
+It would be so sad to tear it down.
+
+She's trying to encourage participation by holding hackathons as part of our team collaborative time and it would be good to roll this out to more of the Institute.
+**Kirstie** agreed that it needs a little more development first.
+Apart from allowing GitHub login it's exactly the same as [mybinder.org](https://mybinder.org/), whereas what people really want or need is to use Binder for private code or data.
+The lab also sent some solidarity - its tough to be the only person working on a tool or infrastructure project 🤗
+
+**Georgia** asked for advice on combining (continuous) open source and (continual) agile workflows.
+The lab are going to dedicate a future lab meeting to this discussion.
+
+**Ang** shared a paper: "Alpha Waves as a Neuromarker of Autism Spectrum Disorder: The Challenge of Reproducibility and Heterogeneity" (doi: [10.3389/fnins.2018.00662](https://doi.org/10.3389/fnins.2018.00662)).
+He was particularly shocked by figure 4 (below).
+**Sarah** recommended [Docker](https://docs.docker.com/engine/docker-overview/) and [Singularity](https://sylabs.io/docs/) as important tools to address this problem.
+They're both described in the _Turing Way_ chapter on [Reproducible Environments](https://the-turing-way.netlify.com/reproducible_environments/06/containers.html).
+Ultimately though, we need a shift in culture that says sharing code and data is no longer enough for reproducibility, but the computational environment the analysis was performed in also needs to be shared.
+
+<figure>
+  <img src="https://www.frontiersin.org/files/Articles/385392/fnins-12-00662-HTML/image_m/fnins-12-00662-g004.jpg" alt="Two scatter plots">
+  <figcaption>Alpha waves characteristics analyzed by Matlab 2013 versus Matlab 2014.
```

Comment:
Do you know which version of the EEGLab toolbox was used in each of these plots? I'm unsure from the text alone whether it's the version of MATLAB causing these differences or if it's also down to the version of the custom toolbox used.